### PR TITLE
Remove s3 for training data

### PR DESCRIPTION
**Pull request by dbason**
Just dump es data and use windows locally

### CODE REVIEW

File: training_controller/prepare_training_logs.py [hunk 1]
```diff
@@ -13,33 +13,22 @@
...
     f"https://{ES_USERNAME}:{ES_PASSWORD}@" + ES_ENDPOINT.split("//")[-1]
 )
+TRAINING_DATA_PATH = os.getenv("TRAINING_DATA_PATH", "/var/opni-data")
 
 
 class PrepareTrainingLogs:
-    def __init__(self, working_dir):
-        self.WORKING_DIR = working_dir
+    def __init__(self):
+        self.WORKING_DIR = TRAINING_DATA_PATH
```
Comment (sanjay920) [inline training_controller/prepare_training_logs.py]: no need to have this be a global var just set to `os.getenv("TRAINING_DATA_PATH", "/var/opni-data")`

**Review by sanjay920 — CHANGES_REQUESTED** [review]

**Comment by tybalex** [comment]
basically it only downloads from ES but no longer upload to S3, am I right?

**Comment by dbason** [comment]
> basically it only downloads from ES but no longer upload to S3, am I right?

Correct

**Review by AmartC — APPROVED** [review]
LGTM

**Review by sanjay920 — APPROVED** [review]

**Pull request merged**